Libraries/Components/ScrollView/RecyclerViewBackedScrollView.android.js

```diff
@@ -113,6 +113,18 @@ var RecyclerViewBackedScrollView = React.createClass({
       );
     });
 
+    const refreshControl = this.props.refreshControl;
+    if (refreshControl) {
+      // Wrap the NativeAndroidRecyclerView with a AndroidSwipeRefreshLayout.
+      return React.cloneElement(
+        refreshControl,
+        {style: props.style},
+        <NativeAndroidRecyclerView {...props}>
```

Comment:
It is getting this style from [here](https://github.com/facebook/react-native/pull/8639/files#diff-d1bfafd1dd132f741cfd85d03201801cR95).
```js
console.log('props.style', props.style[0])
=> props.style Object {flex: 1}